iminuit/tests/test_minimize.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import (absolute_import, division, print_function)
+from iminuit import minimize
+from iminuit.tests.utils import assert_allclose
+import numpy as np
+
+
+def func(x, *args):
+    c = args[0] if args else 1
+    return c + x[0] ** 2 + (x[1] - 1) ** 2 + (x[2] - 2) ** 2
+
+def grad(x, *args):
+    return np.array((2 * x[0], 2 * (x[1] - 1), 2 * (x[2] - 2)))
+
+
+def test_simple():
+    result = minimize(func, (1, 1, 1))
+    assert_allclose(result.x, (0, 1, 2), atol=1e-8)
+    assert_allclose(result.fun, 1)
+    assert(result.nfev > 0)
```

Comment:
assert is a statement, not a function call. Please remove the `()` everywhere you have assert statements.

Reply:
My bad, C++ idioms are leaking in.